knotx-example/knotx-example-app/config/includes/serviceAdapter.conf

```diff
@@ -0,0 +1,59 @@
+# Event bus address of the Basic HTTP Service Adapter
```

Comment:
Do we need here an example (commented out) how to setup connection via SSL?

Reply:
Added more stuff